### ISSUE TITLE
can advanced-cache.php be somewhere else?

### ISSUE THREAD

**Issue by v3nt** [Automattic/batcache#84]
Mainly so it doesn't delete any WP SUPER CACHE files which has the same name in the same place. I know you wouldn't use both but for testing purposes would be good to be able to switch between the 2.

**Comment by johnbillion** [comment]
Nope, the name and location of this file is part of how WordPress itself works, and isn't anything that Batcache, WP Super Cache etc could fix. The same problem occurs with any of the available dropin plugins.
